internal/gen/jsonschema/buf.protoschema.test.v1.ConstraintTest.jsonschema.strict.bundle.json

```diff
@@ -0,0 +1,936 @@
+{
+  "$defs": {
+    "buf.protoschema.test.v1.ConstraintTest.RequiredImplicit.jsonschema.strict.json": {
```

Comment:
In a bundle could this just be the FQN of the protobuf type (without `.jsonschema.strict.json`), or is it required that these are unique across all files?